web/src/app/shared/models/repository.model.ts

```diff
@@ -21,6 +22,7 @@ export class RepositoryModel {
   pullRequests?: PullRequestModel[];
   events?: Event[];
   releases?: ReleaseModel[];
+  issuesList?: IssueModel[];
```

Comment:
I think this does not match with the mapper, in the mapper `issues` is used instead of `issuesList` https://github.com/DashboardHub/PipelineDashboard/pull/1260/commits/61f4c6886bc61ee88c73fa33cac7a0b84c31ded1#diff-9f4cbef5845c995c4f7c82f327df3958R27

My suggestion is keep the mapper how it is, rename it here to `issuesList?: IssueModel[]` -> `issues?: IssueModel[]` and rename the current `issues: number` -> `issueCount: number`

Reply:
Actually, I don't think the original `issues` is used, so just use `issues` instead of `issuesList`

Reply:
I renamed this now and I will test it out

Reply:
I didn't rename this, would it be possible to rename this to `issues`, because I don't think the existing `issues` is used